tensorbay/opendataset/CCPD/catalog.json

```diff
@@ -0,0 +1,34 @@
+{
+    "POLYGON": {
+        "attributes": [
```

Comment:
Are the attributes of CCPD and CCPDGreen the same?

Reply:
yes. The difference between CCPD and CCPDGreen is that the new energy vehicle has an eight-digit license plate number.